packages/react-component-library/src/components/Dropdown/Dropdown.tsx

```diff
@@ -71,7 +71,7 @@ const StyledSelect = styled(Select)`
   }
 
   .rn-dropdown__menu {
-    @include m.shadow('1');
+    box-shadow: 0 4px 16px rgba(0, 0, 0, 0.1), 0 1px 2px rgba(0, 0, 0, 0.1);
```

Comment:
`shadow('1')` is almost invisible and doesn't match Axure or Sketch – I've copied the definition from Sketch instead.